ark/mibi/tiling_utils_test.py

```diff
@@ -465,125 +429,328 @@ def test_create_tiled_regions_non_tma(randomize_setting, moly_run, moly_interval
+    sample_coord = (25000, 35000)
+    new_coord = tiling_utils.convert_microns_to_pixels(sample_coord)
 
-    # easiest case: the center points should be sorted
-    if randomize_setting == ['N', 'N']:
-        assert center_points == actual_center_points_sorted
-    # if there's any sort of randomization involved
-    else:
-        if moly_run == 'N':
-            fov_1_end = 10 if moly_interval_setting else 9
-        else:
-            fov_1_end = 11 if moly_interval_setting else 10
+    assert new_coord == (612, 762)
 
-        # only the second run is randomized
-        if randomize_setting == ['N', 'Y']:
-            # ensure the fov 1 center points are the same for both sorted and random
-            assert center_points[:fov_1_end] == actual_center_points_sorted[:fov_1_end]
 
-            # ensure the random center points for fov 2 contain the same elements
-            # as its sorted version
-            misc_utils.verify_same_elements(
-                computed_center_points=center_points[fov_1_end:],
-                actual_center_points=actual_center_points_sorted[fov_1_end:]
-            )
+def test_assign_closest_fovs():
+    # define the coordinates and fov names generated from the fovd script
+    auto_coords = [(0, 0), (0, 50), (0, 100), (100, 0), (100, 50), (100, 100)]
+    auto_fov_names = ['row%d_col%d' % (x, y) for (x, y) in auto_coords]
 
-            # however, fov 2 sorted entries should NOT equal fov 2 random entries
-            # NOTE: due to randomization, this test will fail once in a blue moon
-            assert center_points[fov_1_end:] != actual_center_points_sorted[fov_1_end:]
-        # both runs are randomized
-        else:
-            # ensure the random center points for fov 1 contain the same elements
-            # as its sorted version
-            misc_utils.verify_same_elements(
-                computed_center_points=center_points[:fov_1_end],
-                actual_center_points=actual_center_points_sorted[:fov_1_end]
+    # generate the list of automatically-generated fovs
+    auto_sample_fovs = dict(zip(auto_fov_names, auto_coords))
+
+    # define the coordinates and fov names proposed by the user
+    manual_coords = [(0, 25), (50, 25), (50, 50), (75, 50), (100, 25)]
+    manual_fov_names = ['row%d_col%d' % (x, y) for (x, y) in manual_coords]
+
+    # generate the list of manual fovs
+    manual_sample_fovs = test_utils.generate_sample_fovs_list(
+        manual_coords, manual_fov_names
+    )
+
+    # generate the mapping from manual to automatically-generated
+    manual_to_auto_map, manual_fovs_info, auto_fovs_info = \
+        tiling_utils.assign_closest_fovs(
+            manual_sample_fovs, auto_sample_fovs
+        )
+
+    # for each manual fov, ensure the centroids are the same in manual_fovs_info
+    for fov in manual_sample_fovs['fovs']:
+        # skip the Moly points
+        if fov['name'] != 'MoQC':
```

Comment:
We got rid of this for the real data, but did we ever have moQC points in the test data?

Reply:
We don't, and since neither `manual` nor `auto` will include any for TMAs, we can remove this test now.